src/genie/libs/parser/iosxr/tests/ShowBfdSession/cli/equal/golden_output1_expected.py

```diff
@@ -0,0 +1,153 @@
+expected_output= {
+    "interface": {
+        "Gi0/0/0/15.103": {
+            "dest_ip_address": {
+                "172.16.100.130": {
+                    "async": "6s",
+                    "async_multiplier": "(2s*3)",
+                    "echo": "450ms",
```

Comment:
same with above.
```suggestion
                    "echo_msec": "450",
```

Reply:
all fields now converted to msec value and include msec in the group name.